src/main/java/jp/cstudio/codereview/SampleCode.java

```diff
@@ -0,0 +1,12 @@
+package jp.cstudio.codereview;
+
+import java.text.SimpleDateFormat;
+
+public class SampleCode {
+    public void sample(){
+        String dateString = "2016/01/01 00:00:00";
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The SimpleDateFormat object was created without setting the time zone. Make sure that you want to use the default time zone or use letter z, Z or X in the pattern. Otherwise, if you do not call setTimeZone() on the created SimpleDateFormat object, you might get an unexpected date and time when using the object.